### PR TITLE
Bioscan EPIC

### PR DESCRIPTION
**Pull request by harrietc52**
Includes
* https://github.com/sanger/limber/pull/1268
* https://github.com/sanger/limber/pull/1269

### CODE REVIEW

File: config/purposes/bioscan.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+LBSN-384 PCR 2:
+  :asset_type: plate
+  # TODO: May need a specific version of TaggedPlate creator to persist tag plate index 1-24
+  :creator_class: LabwareCreators::TaggedPlate
+  :presenter_class: Presenters::MinimalPcrPlatePresenter
+  :label_template: plate_6mm_double
+  :size: 384
+  # TODO check the names for this list of 24 tag layout templates (see tag plate gen stories)
+  :tag_layout_templates:
+    - Bioscan_384_template_1
```
Comment (andrewsparkes) [inline config/purposes/bioscan.yml:54]: This is a problem if we want to introduce new versions. this set should be flagged v2, but any new set is going to require a change to this config and a re-deploy.

File: config/robots.rb [hunk 2]
```diff
@@ -2799,4 +2799,86 @@
...
+        }
+      }
+    )
+
+    custom_robot(
+      'beckman-lilys-96-stock-to-lbsn-96-lysate',
+      name: 'Beckman LILYS-96 Stock => LBSN-96 Lysate',
+      verify_robot: true,
+      beds: {
+        # TODO: confirm beds
```
Comment (sdjmchattie) [inline config/robots.rb:2823]: Does this need a backlog item adding and a reference to it putting here?  Bare TODOs live forever in the code and don't get done.

File: config/robots.rb [hunk 3]
```diff
@@ -2799,4 +2799,86 @@
...
+          child: bed(5).barcode,
+          label: 'Bed 3'
+        },
+        bed(4).barcode => {
+          purpose: 'LBSN-96 Lysate',
+          states: ['passed'],
+          child: bed(5).barcode,
+          label: 'Bed 4'
+        },
+        # TODO: does the order of the parents here define the quadranting behaviour?
```
Comment (sdjmchattie) [inline config/robots.rb:2868]: Should create a backlog item if this TODO is still needed.

**Review by sdjmchattie — APPROVED** [review]
There are a couple of TODOs that I think should be put in the backlog, but otherwise looks good

File: config/purposes/bioscan.yml [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  # TODO: do we want any metadata entry?
+  # :custom_metadata_fields:
+  #   - metadatafieldname
+LBSN-9216 Lib PCR Pool XP:
+  :asset_type: tube
+  :target: MultiplexedLibraryTube
+  :type: IlluminaHtp::MxTubePurpose
+  :creator_class: LabwareCreators::TubeFromTube
+  # TODO: may need a modified tube presenter with paging for samples
+  :presenter_class: Presenters::SimpleTubePresenter
```
Comment (andrewsparkes) [inline config/purposes/bioscan.yml:108]: FinalTubePresenter

File: config/robots.rb (hunk 2 above)
Comment (andrewsparkes) [reply config/robots.rb:2823]: remove comment

File: config/robots.rb (hunk 3 above)
Comment (andrewsparkes) [reply config/robots.rb:2868]: remove comment, it does

File: config/purposes/bioscan.yml (hunk 1 above)
Comment (andrewsparkes) [reply config/purposes/bioscan.yml]: Except there may be 2 sets in progress at a time, e.g. R&D are testing a new version whilst production is on the previous one. So after discussion we decided to build distinctly named tag groups and templates for each version.

**Pull request merged**